app/src/main/java/com/vuongnq14798/vuongmp3/mediaplayer/MediaPlayerManager.kt

```diff
@@ -0,0 +1,140 @@
+package com.vuongnq14798.vuongmp3.mediaplayer
+
+import android.media.AudioAttributes
+import android.media.MediaPlayer
+import android.net.Uri
+import com.vuongnq14798.vuongmp3.data.model.Track
+import com.vuongnq14798.vuongmp3.service.MediaPlayerService
+import com.vuongnq14798.vuongmp3.util.LoopType
+import com.vuongnq14798.vuongmp3.util.ShuffleType
+import com.vuongnq14798.vuongmp3.util.StateType
+import kotlin.random.Random
+
+class MediaPlayerManager(private val mediaPlayerService: MediaPlayerService) :
+    MediaPlayerInterface {
+
+    @StateType
+    var state: Int = StateType.PAUSED
```

Comment:
Nếu chỉ có 1 anno thì để cùng dòng cũng được.
```suggestion
    @StateType var state = StateType.PAUSED
```